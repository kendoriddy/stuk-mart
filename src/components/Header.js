import React from 'react';
import '../css/header.css';
import Search from './Search';

const Header = () => (
  <header>
    <div className="navbar">
      <h3>Stuk-Mart</h3>
    </div>
    <Search />
  </header>
);

export default Header;
