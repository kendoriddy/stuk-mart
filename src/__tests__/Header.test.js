import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Header from '../components/Header';

it('renders correctly', () => {
  const tree = Renderer.create(
    <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <Header />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
