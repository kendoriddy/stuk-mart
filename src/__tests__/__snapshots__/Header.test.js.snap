// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <div
    className="navbar"
  >
    <h3>
      Stuk-Mart
    </h3>
  </div>
  <div
    className="search-name"
  >
    <input
      className="px-2"
      id="search"
      onChange={[Function]}
      placeholder="Search company by name..."
      type="search"
      value=""
    />
  </div>
</header>
`;
